wowup-electron/src/app/pages/options/options.component.html

```diff
@@ -60,6 +60,24 @@
             category="Options" action="UseHardwareAcceleration" [label]="useHardwareAcceleration">
           </mat-slide-toggle>
         </div>
```

Comment:
New switches in the Options view
Ideally, I would make the second one disabled depending on the state of the first one. However, I just did not find a way to do that besides using this thing: https://angular.io/api/forms/NgModel. Yet I did not see anyting like this in the markup, so I decided to leave it as is - this does not really hurt anything. Maybe switches' text can be modified to be more clear in this regard.